Sources/SwiftSyntax/RawSyntax.swift

```diff
@@ -251,9 +221,51 @@ extension RawSyntax {
     return trailingTrivia?.sourceLength ?? .zero
   }
 
-  /// The length of this node including all of its trivia
-  var totalLength: SourceLength {
-    return leadingTriviaLength + contentLength + trailingTriviaLength
+  /// The length of this node excluding its leading and trailing trivia.
+  var contentLength: SourceLength {
+    return totalLength - leadingTriviaLength - trailingTriviaLength
```

Comment:
Even though subtraction is left-associative, I think parens would be nice here just to clarify slightly.
```suggestion
    return totalLength - (leadingTriviaLength + trailingTriviaLength)
```